docs/electron.rst

```diff
@@ -242,3 +242,21 @@ on ``$PATH``:
       # Install app wrapper
       - install -Dm755 -t /app/bin/ ../run.sh
 
+Settings in package.json
+------------------------
+
+If the application that you are trying to package contains a ``build`` block in ``package.json`` with instructions for Linux, this can cause ``electron-builder`` to try to fetch additional binaries at build-time (Even if `--dir` option is used). The following example shows a configuration that will try to download AppImage binaries:
+
+.. code-block:: json
+
+  "build": {
+    "linux": {
+      "target": "AppImage",
+    }
+  }
+
+The preferred way of fixing this, is not a patch, but a build-time edit using `jq`:
```

Comment:
Please add an explanation what this fix does to prevent blind copy-pasting (it sets electron-builder linux build target format to `dir`).

Reply:
Good idea. Done